pages/coinbase/dashboard.py

```diff
@@ -6,9 +6,19 @@
 
 class DashboardPage:
 
-    def __init__(self, driver, drivethrough=True):
+    def __init__(self, driver, spawn=False):
+        """
+        inputs
+        -----
+        driver = selenium driver
+        spawn: bool
+            The default use case is that a driver that is already at the 
```

Comment:
"The default use case is that the driver is already navigated to dashboard before this page object is instantiated"

Reply:
Since this is in every page object and has the same function in every page object, I want to keep it general.